src/unity/lib/visualization/histogram.cpp

```diff
@@ -188,11 +188,21 @@ void histogram_result::add_element_simple(const flexible_type& value) {
    * add element to histogram
    */
 
-  // ignore undefined values
+
   if (value.get_type() == flex_type_enum::UNDEFINED) {
     return;
   }
 
+  if (value.get_type() == flex_type_enum::FLOAT &&
+      std::isinf(value.get<flex_float>())) {
+    return;
+  }
+
+  if (value.get_type() == flex_type_enum::INTEGER &&
+      std::isinf(value.get<flex_int>())) {
```

Comment:
Integers can't be inf or nan (only float). I think you can remove this check.

Reply:
Done!